### PR TITLE
fixes some missing arguments and runtimes

### PR DESCRIPTION
**Pull request by Couls**
## What Does This PR Do

Uses dream checker to fix some missing keywords
Credit to affected for providing the dreamchecker log

## Why It's Good For The Game

better code, less runtimes
## Changelog
:cl:
fix: missing arguments in some procs
fix: some macros that were defined twice
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/inventory.dm [hunk 1]
```diff
@@ -572,5 +572,3 @@
 /mob/living/carbon/human/proc/delete_equipment()
 	for(var/slot in get_all_slots())//order matters, dependant slots go first
 		qdel(slot)
-	for(var/obj/item/I in l_hand, r_hand)
```
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/human/inventory.dm]: are you SURE the above also returns the hands.

File: code/modules/mob/living/silicon/silicon.dm [hunk 2]
```diff
@@ -114,7 +114,7 @@
 
 	return 2
 
-/mob/living/silicon/apply_effect(var/effect = 0,var/effecttype = STUN, var/blocked = 0)
+/mob/living/silicon/apply_effect(var/effect = 0,var/effecttype = STUN, var/blocked = 0, var/negate_armor = 0)
```
Comment (Fox-McCloud) [inline code/modules/mob/living/silicon/silicon.dm]: no implied vars, for petes sake.

File: code/modules/projectiles/guns/energy/kinetic_accelerator.dm [hunk 3]
```diff
@@ -105,7 +105,7 @@
 		empty()
 
 /obj/item/gun/energy/kinetic_accelerator/proc/empty()
-	power_supply.use(500)
+	power_supply?.use(500)
```
Comment (Fox-McCloud) [inline code/modules/projectiles/guns/energy/kinetic_accelerator.dm]: No this is not how you fix this. drop this form this PR, please.

File: code/modules/mob/living/carbon/human/inventory.dm (hunk 1 above)
Comment (Couls) [reply code/modules/mob/living/carbon/human/inventory.dm]: get_all_slots returns l_hand and r_hand

File: code/modules/mob/living/silicon/silicon.dm (hunk 2 above)
Comment (Couls) [reply code/modules/mob/living/silicon/silicon.dm]: this is how it's defined in proc/apply_effect

File: code/_onclick/hud/parallax.dm [hunk 4]
```diff
@@ -310,10 +310,4 @@
 		1, 1, 1, 1,
 		0, 0, 0, 0
 		)
-	screen_loc = "CENTER-7,CENTER-7"
-
-
-#undef LOOP_NONE
-#undef LOOP_NORMAL
-#undef LOOP_REVERSE
-#undef LOOP_TIME
```
Comment (Fox-McCloud) [inline code/_onclick/hud/parallax.dm]: But why?

Comment (Couls) [reply code/_onclick/hud/parallax.dm]: ‘Twas never defined!

**Pull request merged**